llama_stack/providers/inline/inference/vllm/config.py

```diff
@@ -26,34 +24,31 @@ class VLLMConfig(BaseModel):
         default=4096,
         description="Maximum number of tokens to generate.",
     )
+    max_model_len: int = Field(
+        default=4096, description="Maximum context length to use during serving."
+    )
+    max_num_seqs: int = Field(
+        default=4, description="Maximum parallel batch size for generation."
+    )
     enforce_eager: bool = Field(
         default=False,
         description="Whether to use eager mode for inference (otherwise cuda graphs are used).",
     )
     gpu_memory_utilization: float = Field(
         default=0.3,
+        description=(
+            "How much GPU memory will be allocated when this provider has finished "
+            "loading, including memory that was already allocated before loading."
+        ),
     )
 
     @classmethod
     def sample_run_config(cls):
         return {
-            "model": "${env.INFERENCE_MODEL:Llama3.2-3B-Instruct}",
             "tensor_parallel_size": "${env.TENSOR_PARALLEL_SIZE:1}",
             "max_tokens": "${env.MAX_TOKENS:4096}",
+            "max_model_len": "${env.MAX_MODEL_LEN:4096}",
+            "max_num_seqs": "${env.MAX_NUM_SEQS:4}",
```

Comment:
You may need to run distro_codegen.py once to get the updated run.yaml config.

Reply:
Ran distro_codegen.py.